streams/src/test/java/org/apache/kafka/streams/processor/internals/StoreToProcessorContextAdapterTest.java

```diff
@@ -43,63 +45,87 @@ public void setUp() {
         context = StoreToProcessorContextAdapter.adapt(delegate);
     }
 
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     public void shouldThrowOnCurrentSystemTime() {
-        context.currentSystemTimeMs();
+        assertThrows(UnsupportedOperationException.class, () -> {
+            context.currentSystemTimeMs();
+        });
```

Comment:
We don't need to include the statement in curly braces given this is a single line. Typically you will have IDEs throwing warnings like 
`Statement lambda can be replaced with expression lambda `
This comment would be applicable to all changed lines.

Reply:
Thanks for pointing this out. I agree. I will fix this shortly.

Reply:
Thank you 👍